api/src/main/java/org/openmrs/module/idgen/LogEntry.java

```diff
@@ -20,36 +20,37 @@
  * Component which encapsulates a Log entry
  */
 public class LogEntry {
-	
+
 	//***** PROPERTIES *****
-	
-    private Integer id;
-    private IdentifierSource source;
-    private String identifier;
-    private Date dateGenerated;
-    private User generatedBy;
-    private String comment;
-	
-    //***** CONSTRUCTORS *****
-    
```

Comment:
Why did you remove these?
The only change i expected was an addition of a getUuid() method to this class.

Reply:
@dkayiwa, I didnt remove anything, just added the method as requested. Probably its the editor formatting making it look that way but all the code is still there as shown on the left side of the screenshot
<img width="1248" alt="screen shot 2017-10-09 at 1 52 43 pm" src="https://user-images.githubusercontent.com/26176338/31335016-60ffb3f2-acf9-11e7-817f-d3df470353c2.png">

Reply:
Can you revert the changes in it. Then just add the new method and commit?

Reply:
Reverted the changes :)